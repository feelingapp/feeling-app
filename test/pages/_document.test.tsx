import { shallow } from "enzyme"
import MyDocument from "../../src/pages/_document"

describe("Document: ", () => {
  it("matches snapshot", () => {
    const wrapper = shallow(<MyDocument />)
    expect(wrapper).toMatchSnapshot()
  })
})
