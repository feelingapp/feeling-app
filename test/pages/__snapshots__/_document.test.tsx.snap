// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Document:  matches snapshot 1`] = `
<Html
  lang="en"
>
  <Head>
    <link
      href="/icons/16.png"
      rel="icon"
      sizes="16x16"
      type="image/png"
    />
    <link
      href="/icons/32.png"
      rel="icon"
      sizes="32x32"
      type="image/png"
    />
    <link
      href="/icons/apple-touch-icon.png"
      rel="apple-touch-icon"
      sizes="180x180"
    />
    <link
      color="#caeaf8"
      href="/icons/safari-pinned-tab.svg"
      rel="mask-icon"
    />
    <link
      href="/manifest.json"
      rel="manifest"
    />
    <meta
      content="#fefefe"
      name="theme-color"
    />
  </Head>
  <body>
    <Main />
    <NextScript />
  </body>
</Html>
`;
